script/version.json

```diff
@@ -1,3 +1,3 @@
 {
-  "version": "0.14.5"
+  "version": "0.14.6"
```

Comment:
not necessary to increment this if we're not changing java or python i dont think

Reply:
We should change it every time.